ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-1.9-release-4.19.yaml

```diff
@@ -26,8 +26,8 @@ tests:
       LOKI_ENABLED: "false"
       SLEEP_DURATION: 2h
       SPOT_INSTANCES: "false"
-      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&;C00102
-      TEST_SCENARIOS: kata
+      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
+      TEST_SCENARIOS: C00102
```

Comment:
Could you please improve the commit message? IIUC this is changing the suite definition, removes the filter-only C00102 test and adds it using TEST_SCENARIOS (which is indeed more appropriate). Also perhaps this can be squashed to previous commit as we are introducing this, what do you think?